web/packages/teleterm/src/ui/AccessRequestCheckout/useAccessRequestCheckout.test.tsx

```diff
@@ -230,14 +230,14 @@ test('after creating an access request, pending requests and specifiable fields
     wrapper,
   });
 
-  await act(() =>
+  act(() =>
```

Comment:
i dont understand how this passed 🤔 (i tried removing the awaits...)

Reply:
It's pretty much the same thing that I mentioned in the comment that I link to in the PR description. I bet if you tried to remove this and only this `await` in your original version, it'd work too because `setSelectedResourceRequestRoles` doesn't cause that resource roles effect to re-run.